static/workspace-view.less

```diff
@@ -22,7 +22,9 @@ atom-workspace {
   position: relative;
   color: @text-color;
   background-color: @app-background-color;
-  font-family: @font-family;
+  font: menu;
```

Comment:
Ah okay. I think we need to keep `font-size` and `line-height` though, because interestingly enough the `menu` shorthand also defines new a new `font-size` and `line-height`, not just a new `font-family`